error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StringRules.java

```diff
@@ -162,6 +162,22 @@ String after(Object object) {
     }
   }
 
+  /**
+   * Prefer direct invocation of {@link String#copyValueOf(char[])} over the indirection introduced
+   * by {@link String#copyValueOf(char[])}.
+   */
+  static final class StringCopyValueOf {
+    @BeforeTemplate
+    String before(char[] data, int offset, int count) {
+      return String.copyValueOf(data, offset, count);
```

Comment:
I see that we can also rewrite a `String.valueOf` variant; will propose something.